reana_commons/openapi_specifications/reana_server.json

```diff
@@ -2768,4 +2776,4 @@
   },
   "swagger": "2.0",
   "tags": []
-}
+}
```

Comment:
Missing newline

Reply:
Thanks for spotting, it's not the first time it happens to me and it's a bit annoying. I think we could modify our `generate_openapi_spec.py` script to include new line for us to avoid these kind of issues in the future

Reply:
Created [a new commit](https://github.com/reanahub/reana-server/pull/433/commits/909e4c61de7a9fc2e026a8cd40472dea7a555647) in `reana-server` and a [new PR](https://github.com/reanahub/reana-workflow-controller/pull/431) in `reana-workflow-controller` to fix it